cypress/integration/smoke/balances.spec.js

```diff
@@ -0,0 +1,196 @@
+const assetsTableContainer = '.MuiTableContainer-root'
+const balanceRowTestId = '[data-testid=balance-row]'
+const receiveModalClass = '.receive-modal'
+
+const TEST_SAFE = '/rin:0x11Df0fa87b30080d59eba632570f620e37f2a8f7'
+const ASSETS_LENGTH = 7
+
+describe('Assets > Coins', () => {
+  const TOKEN_AMOUNT_COLUMN = 1
+  const FIAT_AMOUNT_COLUMN = 2
+
+  // Fiat balance regex
+  const fiatRegex = new RegExp(`([0-9]{1,3},)*[0-9]{1,3}.[0-9]{2}`)
+  // Token balance regex
+  const tokenRegex = new RegExp(`([0-9]{1,3},)*[0-9]{1,3}.[0-9]{2,5}`)
+
+  before(() => {
+    // Open the Safe used for testing
+    cy.visit(`${TEST_SAFE}/balances`)
+    cy.contains('a', 'Accept selection').click()
+  })
+
+  describe('should have different tokens', () => {
+    it(`should have ${ASSETS_LENGTH} entries in the table`, () => {
+      cy.get(balanceRowTestId).should('have.length', ASSETS_LENGTH)
+    })
+
+    it('should have Dai', () => {
+      const DAI_ROW_NUMBER = 0
+      // First row should be Dai
+      cy.get(balanceRowTestId).eq(DAI_ROW_NUMBER).contains('Dai')
```

Comment:
```suggestion
      const daiRow = cy.get(balanceRowTestId).eq(DAI_ROW_NUMBER)
      daiRow.contains('Dai')
```

Reply:
I was trying to abstract `cy.get(balanceRowTestId).eq(DAI_ROW_NUMBER)` to the variable like you suggest and use it in the further assertions but it seems to not be equivalent to write the `get` explicitly.
I will leave it as it is.